test/test.js

```diff
@@ -0,0 +1,46 @@
+var expect  = require("chai").expect;
+var request = require("request");
+var assert = require('assert');
+
+
+describe('Routes', function() {
+
+  var url = "http://localhost:3000/";
+
+
+  describe('Search Video DB', function() {
```

Comment:
I would recommend being a little more descriptive, what are you expecting searching video db for .....